snuba/migrations/snuba_migrations/replays/0001_replays.py

```diff
@@ -0,0 +1,96 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    UUID,
+    Array,
+    Column,
+    DateTime,
+    IPv4,
+    IPv6,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+raw_columns: Sequence[Column[Modifiers]] = [
+    Column("replay_id", UUID()),
+    Column("sequence_id", UInt(16)),
```

Comment:
Is there any sort of relation between `sequence_id` and `replay_id` field?

Reply:
`sequence_id` will be a monotonically increasing counter, so for each `replay_id`, `sequence_id` is unique.

Reply:
Will `sequence_id` always start from a fixed number for each `replay_id`? And how many `sequence_id` are allowed for a single `replay_id`. Is there a maximum?